index.html

```diff
@@ -401,6 +401,9 @@ <h3>SigLens v/s ClickHouse</h3>
           <div id="docker" role="tabpanel" class="tab-pane active">
             <div class="code position-relative"><span id="install-text">curl -L https://siglens.com/install.sh | sh</span>
               <i class="fa-regular fa-clone position-absolute copy-btn" id="copyBtn" data-toggle="tooltip" ></i>
+              <svg class="check-mark" id="check-mark" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Add the check  svg file locally in assets.

Reply:
This svg is an animation. The <img> tag of html does not support animation inside SVG, so we can only embed SVG code into the HTML document.

Reply:
sorry, missing words: The <img> tag of html does not support animation inside SVG, so we can only embed SVG code into the HTML document.